src/app/frontend/common/dialogs/editresource/template.html

```diff
@@ -30,7 +30,7 @@ <h2 mat-dialog-title>Edit a resource</h2>
     <mat-icon>info</mat-icon>
     <div>
       <span>This action is equivalent to:</span>
-      <code>kubectl apply -f &lt;spec.json&gt;</code>
+      <code>kubectl apply -f &lt;spec.{{selectedMode_}}&gt;</code>
```

Comment:
When trying to run this locally I get:

```
[1] ERROR in src/app/frontend/common/dialogs/editresource/template.html(33,13): : Property 'selectedMode_' is private and only accessible within class 'EditResourceDialog'.
[1] ℹ ｢wdm｣: Failed to compile.
```

Which is true, `selectedMode_` is [declared as private](https://github.com/kubernetes/dashboard/blob/master/src/app/frontend/common/dialogs/editresource/dialog.ts#L33)

Reply:
so should i make another public variable for this?

Reply:
You should change existing one to public and rename it to remove `_` from name.

Reply:
ok thanks